frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -124,9 +127,15 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
   const handleModeChange = (event: React.FormEvent<HTMLInputElement>) => {
     const { value } = event.currentTarget;
     const modeIndex = getIndex(MODES, value);
-    history.push(
-      `~new?${getParamString(getIndex(CreateStepsSC, CreateStepsSC.DISCOVER), modeIndex)}`,
-    );
+    switch (modeIndex) {
+      case 2:
+        if (hasNoProvSC) {
+          history.push(getAnchor(3, 2));
+        }
+        break;
```

Comment:
Shouldn't this fall back to default?  Maybe the break should be part of the above `if` block?